static/js/constants.js

```diff
@@ -103,6 +108,9 @@ function getIcon(accidentType, severity) {
     try {
         if (accidentType == "multiple") {
             icon = MULTIPLE_ICONS[severity];
+
+        } else if (accidentType == ACCIDENT_TYPE_UNITED_HATZALA) {
+            icon = ICONS[severity][ACCIDENT_TYPE_UNITED_HATZALA];
```

Comment:
Why not just add an entry to `ACCIDENT_MINOR_TYPE_TO_TYPE` rather than adding an `if`?